### DISCUSSION TITLE
Enhancement Proposal: Title Attribute for RemoteItem

### DISCUSSION THREAD

**Discussion by nathangathright** [Podcastindex-org/podcast-namespace#629] [Enhancement Proposal]
After looking at the [publisher feeds proposal](https://github.com/Podcastindex-org/podcast-namespace/blob/main/publishers/publishers.md), I believe the `<remoteItem>` tag would benefit from the addition of an optional title attribute.

Obviously, a canonical `<title>` is accessible if one goes to the trouble of scraping the remote content in question, but for some use cases, that’s a rather high barrier to entry. A title attribute on the `<remoteItem>` tag would provide useful context before accessing that content.

This does create the risk of mismatches between the `title` attribute and the remote `<title>` tag. As time goes on, remote content (podcasts, publishers, bands, episodes, songs) may change their name. But similar to `<a href="">Inner Text</a>` in links, this is a small downside to provide some amount of context.

### Publisher Feeds
On Episodes.fm, I currently display the `<itunes:author>` tag below the podcast name. It would be nice to replace this with a link to a publisher page, and pulling a title from an attribute would be more appropriate than using the `<itunes:author>` value.

### Podroll
Same for Podroll, a title attribute would allow apps enough additional context to display a link without scraping the remote content in advance.

**Comment by daveajones** [comment]
Thanks.  Linking to Phase 7 plan.

**Comment by daveajones** [comment]
This has been added.
